src/app/static/src/app/store/adr/actions.ts

```diff
@@ -67,7 +90,7 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
 
         if (selectedDataset) {
             if (!selectedDataset.organization) {
-                await dispatch("getAndSetDatasets", selectedDataset.id);
+                await dispatch("getDataset", {id: selectedDataset.id, release: selectedDataset.release});
```

Comment:
With newDatasetVersionId hard-coded to "1.0", I ran a model up to the Review outputs step, then reloaded the page, then went on to the Downloads step. I saw the dispatched call from this line to get the dataset details but did not see the "release=1.0" param on the url as I did when the dataset details were initially fetched. It doesn't really matter, since releases don't have different permissions, but I did expect that release to have been persisted...

Reply:
ba9b05d